core/sentry.js

```diff
@@ -0,0 +1,112 @@
+/* @flow */
+
+const Raven = require('raven')
+const logger = require('./logger')
+const log = logger({
+  component: 'Sentry'
+})
+
+const { omit } = require('lodash')
+
+const DOMAIN_REGEXP = /^https?:\/\/([a-zA-Z0-9-.]+)(?::\d{2,5})?\/?$/
+const SENTRY_REF = `ed6d0a175d504ead84851717b9bdb72e:324375dbe2ae4bbf8c212ae4eaf26289`
```

Comment:
Are Cozy webapps refs embedded like this too? I mean, is there a way to prevent anybody from spamming our Sentry without being identified easily?

Reply:
Yes and No

Reply:
:ok_hand: